pkg/util/validation/limits.go

```diff
@@ -457,6 +460,15 @@ func (o *Overrides) MaxQueryLength(userID string) time.Duration {
 	return time.Duration(o.getOverridesForUser(userID).MaxQueryLength)
 }
 
+// MaxPartialQueryLength returns the limit of the length (in time) of a (partial) query.
+func (o *Overrides) MaxPartialQueryLength(userID string) time.Duration {
+	t := time.Duration(o.getOverridesForUser(userID).MaxPartialQueryLength)
+	if t == time.Duration(0) {
+		t = o.MaxQueryLength(userID)
```

Comment:
I would unexport `MaxQueryLength` because now we expect none to use it from other packages.